### PR TITLE
playbook #177: Suggestion to make peer review FED process more inclusive

### PR DESCRIPTION
**Pull request by Alicole**
playbook #177: Suggestion to make peer review FED process more inclusive

### CODE REVIEW

**Comment by andrewmee** [comment]
Could perhaps be simplified?

"Ensure that any changes to front-end code conforms to the same standards and guidelines."

File: practices/code-review.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ We manage code review via **pull requests**, which we communicate in the `#front
     * allows more junior members of staff to learn best practices quicker.
     * allows EVERYONE to learn from their colleagues.
     * facilitates people moving to other teams as they are more familiar with the codebase.
-* Ensure that Front-End Developers within each team are coding to the same guidelines.
+* Ensure that any developers, within each team, who are changing front-end templates or code are conforming to the same guidelines.
```
Comment (jpw) [inline practices/code-review.md]: the whole sentence can be deleted IMO, it's covered by "allows for standardisation of methodology across teams", above

Comment (Alicole) [inline practices/code-review.md]: Which is it to be remove or edit?

Comment (andrewmee) [inline practices/code-review.md]: I think that point is just relating to the fact that it's cross-team (although I agree there's a good deal of overlap)

Comment (Alicole) [inline practices/code-review.md]: To be honest we don't make it clear why we want devs to carry out PRs following front-end practices. Other dev teams don't always follow this process, and some find it rather fussy.

**Pull request merged**